src/lib/src/viewer/viewer.component.spec.ts

```diff
@@ -346,7 +345,7 @@ describe('ViewerComponent', function () {
 
 
   it('should change page when swipeing to left', () => {
```

Comment:
Should this test be moved to ModeService?

Reply:
I have now deleted these tests as they were quite similar to some e2e tests. Then I added some new unit tests for the ModeService.